doc/news/_preparation_next_release.md

```diff
@@ -139,7 +147,24 @@ you up to date with the multi-language support provided by Elektra.
 
 - Remove `-Wno-shift-overflow warnings` option from SWIG bindings compile flags. _(Mihael Pranjić)_
 
-### <<Binding2>>
+### JNA
+
+- Gradle wrapper upgraded to 7.0
+- Updated documentation
+- Increased minimum required JDK version to 9
+- Migrated native resource clean-up from `finalize()` to `Cleaner`
+- Extracted exceptions from Key class introducing the following changes
+  - Removed unused `KeyTypeConversionException`
+  - Introduced KeyCreateFailedException, KeyReleasedException
+  - Renamed `KeyInvalidNameException` to `KeySetNameFailedException`
+  - Renamed `KeyTypeMismatchException` to `KeyBinaryTypeNotSupportedException`
+- Introduced `KeySetReleasedException`
+- `KeySet::lookup*` now returns `Optional<Key>`
+- `Key::getMeta` now returns `Optional<Key>`
+- Removed `Key::isNull`
```

Comment:
Write what needs to be done instead now, give code examples and so on.

Reply:
release notes updated

I do not think examples regarding usage or Java optionals are appropriate. Are there any specific examples you would want to see?

Reply:
Not "I want to see" but think of users of your binding. If someone already uses `lookup` or `getMeta` he/she wants to know how to adapt the existing code. The easiest way to see that is with an example.

Reply:
I see your point, even though the API change is trivial. Further, there are many different ways, how to process the resulting Optional, depending on the callers structure and the context, the method result is to be used...
I've added an example to the changelog.

Reply:
It is essential that it is always clear that every change somehow improves the user experience. This should be clear everywhere: in the code, in the release notes and also in the bachelor thesis :heart: